src/logos.ts

```diff
@@ -30,6 +30,12 @@ export const apple: IMXIcon = {
   categories: ["logos"],
 };
 
+export const battlenet: IMXIcon = {
```

Comment:
Please, also add this to the logos array at the bottom of the document.

Reply:
Done